### PR TITLE
Format readme

### PR DESCRIPTION
**Pull request by vladimyr**
Format html & markdown content. Use https://badgen.net as badge provider. :rocket:

### CODE REVIEW

**Comment by vladimyr** [comment]
If you wan't I can switch bundlephobia badge with min+gzip one so it matches previous advertised size: https://bundlephobia.com/result?p=vue-gtag@1.1.1

File: README.md [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-
- <img src="https://img.shields.io/badge/size-2.07kB-brightgreen.svg" />
- 
+  <img width="250" src="https://i.imgur.com/AlAf04U.png">
+  <br>
+  <a href="https://npm.im/vue-gtag">
+    <img src="https://badgen.net/npm/v/vue-gtag">
+  </a>
+  <a href="https://bundlephobia.com/result?p=vue-gtag">
+    <img src="https://badgen.net/bundlephobia/min/vue-gtag">
```
Comment (MatteoGabriele) [inline README.md]: i'd use the min gzip, most of the time that's the file size that people look at

`https://badgen.net/bundlephobia/minzip/vue-gtag`

Comment (vladimyr) [reply README.md]: Sure, I'll update it.

**Pull request merged**